### PR TITLE
initialized ticket model

### PR DESCRIPTION
**Pull request by elvisEspinozaN**
attempt #2

### CODE REVIEW

**Review by JackTVanDyke — APPROVED** [review]
I see you went with UUID, I'll have to read up on it. There's so much stuff to learn ffs

File: src/main/java/genspark/group/boardingpass/Ticket.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        this.boardingPassNumber= (generateBoardingPassNumber());
+        this.eta= generateEta(this.departureTime);
+        this.price= generatePrice(this.age, this.gender);
+    }
+
+    private UUID generateBoardingPassNumber() { // attempting to use UUId and inserted MAX_VALUE to force positive
+        return UUID.randomUUID();
+    }
+
+    private Date generateEta(Date departureTime) { // TODO - time math
```
Comment (spencerk80) [inline src/main/java/genspark/group/boardingpass/Ticket.java:49]: This method doesn't do anything, but I see there's a to-do comment, so don't forget to update this

File: src/main/java/genspark/group/boardingpass/Ticket.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                ", boardingPassNumber=" + boardingPassNumber +
+                ", date='" + date + '\'' +
+                ", origin='" + origin + '\'' +
+                ", destination='" + destination + '\'' +
+                ", departureTime='" + departureTime + '\'' +
+                ", estimatedTimeOfArrival='" + eta + '\'' +
+                ", ticketPrice=" + price +
+                '}';
+    }
+}
```
Comment (spencerk80) [inline src/main/java/genspark/group/boardingpass/Ticket.java:126]: TO-DO: Please auto-generate a hash and equals

**Review by spencerk80 — APPROVED** [review]
There are a couple of to-do items, but otherwise it looks good. Will approve, but I'm going to open tickets for the tasks to be completed

**Comment by gensparkweeks** [comment]
Aprroved

**Review by gensparkweeks — APPROVED** [review]
ok

**Review by spencerk80 — APPROVED** [review]
Looks great!
Maybe there's an issue. Whatever

**Pull request merged**